content/800-data-platform/200-pulse/200-getting-started.mdx

```diff
@@ -20,16 +20,98 @@ To participate in Pulse's Early Access program, you need to meet the following p
 
 You will also need a database with the following configurations:
 
-- Postgres version 12+.
+- PostgreSQL version 12+.
 - Ensure your database is publicly accessible.
-- Enable [logical replication](https://www.postgresql.org/docs/current/logical-replication-quick-setup.html) on your database.
+- [Set the `wal_level` setting in PostgreSQL to `logical`](/data-platform/pulse/getting-started#wal_level)
 - A database superuser that can be used for connections inside Pulse.
 
-## 1. Enable Pulse in a Cloud Project
+## 1. Database setup
+
+Depending on your database provider, the setup may vary. This section will attempt to cover the most common providers. If you have a provider that isn't covered, please reach out to a developer advocate at Prisma for assistance.
+
+### General database configuration
+
+#### <inlinecode>[wal_level](https://www.postgresql.org/docs/current/runtime-config-wal.html)</inlinecode>
+
+Some providers may not allow direct access to this setting. If you are unable to change this setting, please refer to the provider-specific guides for further assistance.
+
+```SQL
+  ALTER SYSTEM SET wal_level = logical;
+```
+
+You will need to restart the database after changing this setting.
+
+#### Optional settings
+
+The following increases the memory usage of the [write-ahead log](https://www.postgresql.org/docs/current/wal-intro.html) on your PostgreSQL database. We suggest setting these values initially and adjusting them if necessary.
+
+#### <inlinecode>[max_replication_slots](https://www.postgresql.org/docs/current/runtime-config-replication.html)</inlinecode>
+
+```SQL
+  ALTER SYSTEM SET max_replication_slots = 20;
+```
+
+#### <inlinecode>[wal_keep_size](https://www.postgresql.org/docs/current/runtime-config-replication.html)</inlinecode>
+
+```SQL
+  ALTER SYSTEM SET wal_keep_size = 2048;
+```
+
+### Provider specific configuration
+
+#### Railway
+
+[Railway.app](https://railway.app) offers an excellent templates feature. If you wish to quickly start with Pulse, you can utilize our [Prisma Pulse DB Only](https://railway.app/template/pulse-pg) template, which provides a fresh pre-configured PostgreSQL database. Or you can use our [Prisma Pulse DB & App](https://railway.app/template/pulse-starter) railway template that provides a pre-configured PostgreSQL database and a Pulse starter app.
+
+<details><summary>1. Change the PostgreSQL database settings</summary>
+
```

Comment:
I mean, thats really up to them no? There is like 20 different ways they could do it. I can add a "run in railway's ui or how ever you see fit" etc. Just pointing this out cause it could get convoluted fast

Reply:
Yes, I think we should mention Railway UI as the default (ideally even with a link) that users can click to land at the place where they can link it. We can then say that it's also possible using any other SQL client.

Reply:
![image](https://github.com/prisma/docs/assets/36578389/13e1fd67-bca7-490e-96f0-cbb6370fbe22)

It looks ugly, but it'll do. The <details> block makes adding things like block quotes, not work.